Source/WebCore/accessibility/AccessibilityRenderObject.cpp

```diff
@@ -2127,15 +2127,20 @@ AccessibilityRole AccessibilityRenderObject::determineAccessibilityRole()
     if ((m_ariaRole = determineAriaRoleAttribute()) != AccessibilityRole::Unknown && !shouldIgnoreAttributeRole())
         return m_ariaRole;
 
-    if (m_renderer->isRenderListItem())
-        return AccessibilityRole::ListItem;
+    RefPtr node = m_renderer->node();
+    if (m_renderer->isRenderListItem()) {
+        // Active summaries are handled by AccessibilityNodeObject::determineAccessibilityRoleFromNode().
+        RefPtr summary = dynamicDowncast<HTMLSummaryElement>(node);
+        if (!summary || !summary->isActiveSummary())
```

Comment:
Could we please add a comment here saying something to the effect of "The details / summary disclosure triangle is implemented using RenderListItem, so check to make sure we don't have one of those." Re-word as you see fit.

Reply:
Will do, once I fix all the EWS failures

Reply:
Fixed!